go/vt/sqlparser/sql.y

```diff
@@ -2080,9 +2310,21 @@ show_statement:
   {
     $$ = &Show{Type: string($2)}
   }
-| SHOW PROCEDURE ddl_skip_to_end
+| SHOW PROCEDURE CODE ddl_skip_to_end
```

Comment:
Do you actually want the skips on these? Typically those are for unsupported / semi-supported features

Reply:
We don't support the `CODE` ones for now.